### ISSUE TITLE
Feature Request: Add Geo Filters in the Classified Sidebar

### ISSUE THREAD

**Issue by ichpen** [gfnlabs/classifieds#9]
At minimum add filters for:
1. Country
2. State/Province
3. City
4. Proximity i.e. within xxx kms of postal code (if you want to get fancy)

These should be in the sidebar on the classified page.

**Comment by sikhlana** [comment]
@BigIron5 and I discussed on the possibility of how the location will be handled.
I think the best course for this would be something along the line of having a dedicated page where classifieds will be listed based on the GeoIP of the visitor instead of just adding a "Filters" block.

**Comment by ichpen** [comment]
Great idea except you can't use the visitor GeoIP without their consent which means you now need a consent checkbox and phrase. 

Filters box is what people use when going to ebay/kijiji/craigslist. It's something they're used to.

If you're going with GeoIP you may as well have a classified MAP with plotted ads, now that would be cool but different feature.

**Comment by sikhlana** [comment]
Yes, we will have a consent form when the user first visits the page.
I am still open to how these should be handled as this module will take some time for me to develop to something usable.

**Comment by ichpen** [comment]
My vote goes to a locale filter sidebar box, it should hopefully be easier to implement.

Here's an example of what a simplified filter would look like, look at the left sidebar:
http://www.kijiji.ca/b-phone-tablet/ontario/c132l9004
